requirements/django.in

```diff
@@ -1,6 +1,6 @@
-# Requirements for test runs utilizing Django support
+# Requirements for [django] extra, not including base deps
```

Comment:
Comment is a little confusing given that you pull in base below.

Maybe something like?:
```
# Requirements for [django] extra
# Used in setup.py's `extras_require` and not `install_requires`.
```

Reply:
It's using `-c` to constrain the deps, rather than `-r` to pull them in.

Reply:
Consider adding ^ above that line.  For example:
```
# Note:  using -c to constrain the deps, rather than -r to pull them in.
```

Reply:
Added a note here too.